apps/reference/nav/supabase_dart_sidebars.js

```diff
@@ -14,10 +14,14 @@ const sidebars = {
       label: 'Auth',
       items: [
         'generated/auth-signup',
-        'generated/auth-signin',
-        'generated/auth-signinwithprovider',
+        'generated/auth-signinwithpassword',
+        'generated/auth-signinwithotp',
+        'generated/auth-signinwithoauth',
         'generated/auth-signout',
-        'generated/auth-update',
+        'generated/auth-verifyotp',
+        'generated/auth-currentsession',
+        'generated/auth-currentuser',
+        'generated/auth-updateuser',
```

Comment:
@dshukertjr are you seeing this issue when running the dev server locally?
```
[INFO] Starting the development server...
[ERROR] Loading of version failed for version current
[ERROR] Error: Invalid sidebar file at "nav/supabase_dart_sidebars.js".
These sidebar document ids do not exist:
- generated/auth-signinwithoauth
- generated/auth-signinwithotp
- generated/auth-signinwithpassword
- generated/auth-updateuser
- generated/auth-verifyotp
```

Reply:
I couldn't reproduce it, but it seems like after fixing the broken links you referenced below, build is working.